arches_controlled_lists/src/arches_controlled_lists/components/editor/ImageMetadata.vue

```diff
@@ -216,25 +226,22 @@ const setRowFocus = (event: DataTableRowEditInitEvent) => {
     rowIndexToFocus.value = event.index;
 };
 
-const inputSelector = computed(() => {
-    return `[data-p-index="${rowIndexToFocus.value}"]`;
-});
+const makeRowUneditable = (metadataId: string) => {
```

Comment:
nit: function declaration instead

Reply:
for sure, wasn't sure if was better to blend in w/ rest of the file. will do.